coordinator/src/routes.rs

```diff
@@ -117,6 +118,7 @@ pub fn router(
         .route("/api/trade", post(post_trade))
         .route("/api/rollover/:dlc_channel_id", post(rollover))
         .route("/api/register", post(post_register))
+        .route("/api/fcm_token", post(post_fcm_token))
```

Comment:
We really should start authenticating the user.

https://github.com/get10101/10101/issues/1255

Reply:
I don't disagree with your statement, but how is this related to this particular ticket? 🤔

are you suggesting that this PR makes the security situation any worse?

if yes, I'd like to explain that, unline some of the other endpoints, this one has *minimal* security implications. if someone happens to spoof an FCM token they'd get a push notification not meant for them in the worst case.

Reply:
Just referring the ticket so we don't forget to tackle this as well.